packages/terra-application-name/src/ApplicationMenuName.module.scss

```
@@ -1,4 +1,8 @@
 /* stylelint-disable selector-class-pattern */
+// Themes
+@import './clinical-lowlight-theme/ApplicationName.module';
+@import './orion-fusion-theme/ApplicationName.module';
```

```suggestion
@import './clinical-lowlight-theme/ApplicationMenuName.module';
@import './orion-fusion-theme/ApplicationMenuName.module';
```
These theme vars should be located in their corresponding theme files

---

Done 973c836